src/services/RTC.ts

```diff
@@ -11,22 +13,55 @@ class RTC {
     receiveChannel: RTCDataChannel | null;
     customSendChannelStatusHandler: any;
     customReceiveChannelStatusHandler: any;
-    customReceiveDataHandler: any;
+    customReceiveDataHandler: (event: MessageEvent) => void;
     attemptReconnectInterval: any;
     numReconnectRetries: number;
     roomid: string;
 
+    // file transfer fields
+    fileReader: FileReader;
```

Comment:
Is there a specific reason why we have file transfer fields inside this module and also in `FileTransfer.tsx`?

Reply:
I think it's mainly cause originally all the logic was in `FileTransfer.tsx`, and I only recently added the file transfer handling in this class. I'll look into it next PR. I was gonna go back and delete everything we're not using, bc there's kind of alot of unused code out there right now I think.